utils/useVersionAgnosticPages.ts

```diff
@@ -1,6 +1,19 @@
 import { getPathWithoutVersion } from "./url";
 
-const agnost = ["preview/upcoming-releases/", "changelog/"];
+// Paths including these segments are version agnostic. If a URL path includes
+// one of these segments when a user visits a page via a navigation link:
+// - The docs site will render the default version of the page
+// - The version switcher will be disabled on the page
+const agnost = [
+  "upcoming-releases/",
+  "changelog/",
+  "contributing/documentation/",
+  "contributing/documentation/reviewing-docs/",
+  "contributing/documentation/issues/",
+  "contributing/documentation/style-guide/",
+  "contributing/documentation/reference/",
+  "contributing/documentation/how-to-contribute/",
```

Comment:
hmm... i could see this getting out of hand in the future. This works for now but perhaps I'll revisit how we determine if the url is agnostic

Reply:
Sounds good!